ranger/default.go

```diff
@@ -341,35 +338,14 @@ func DefaultRouter() RangerOption {
 			mws = append(
 				mws,
 				middleware.InjectSession(rng.sessions, rng.kr.SessionKey()),
+				middleware.CurrentUser(
+					rng.Responder,
+					rng.users,
```

Comment:
This naming has always thrown me off. I assume it's an array of users or something. Maybe `userFetcher` or something more explicit will improve the readability here.